cmd/cli/plugin/feature/deactivate_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/spf13/cobra"
+)
+
+func TestExecuteFeatureDectivateCommand(t *testing.T) {
+	test := struct {
+		description string
+		cmd         *cobra.Command
+		featName    string
+		errMsg      string
+	}{
+		description: "FeatureGateClient cannot connect due to bad configuration",
```

Comment:
can you add a testcase for feature deactivation working successfully?

Reply:
I added a few more test cases for deactivating a feature.